osidb_bindings/session.py

```diff
@@ -71,6 +71,44 @@ def file_trackers(self, form_data: Dict[str, Any], *args, **kwargs):
     )
 
 
+def promote_flaw(self, id, *args, **kwargs):
+    method_module = importlib.import_module(
+        f".bindings.python_client.api.osidb.osidb_api_{OSIDB_API_VERSION}_flaws_promote_create",
+        package="osidb_bindings",
+    )
+    sync_fn = get_sync_function(method_module)
+    return sync_fn(
+        id,
+        *args,
+        client=self.client(),
+        **kwargs,
+    )
+
+
+def reject_flaw(self, id: str, form_data: Dict[str, Any], *args, **kwargs):
+    method_module = importlib.import_module(
+        f".bindings.python_client.api.osidb.osidb_api_{OSIDB_API_VERSION}_flaws_reject_create",
+        package="osidb_bindings",
+    )
+    sync_fn = get_sync_function(method_module)
+    model = getattr(method_module, "REQUEST_BODY_TYPE", None)
+    if model is None:
+        raise UndefinedRequestBody(
+            f'The request body for the resource "flaw" '
+            f'and the operation "reject" is not defined.'
+        )
+    transformed_data = model.from_dict(form_data)
```

Comment:
Actually there is a [new safer function](https://github.com/RedHatProductSecurity/osidb-bindings/blob/master/osidb_bindings/session.py#L115) for serializing the data which should be preferred. Would you mind fixing this also in the [`file_trackers`](https://github.com/RedHatProductSecurity/osidb-bindings/blob/master/osidb_bindings/session.py#L48) function as I've probably forgot about that there?